src/components/multi-action-button/multi-action-button.spec.tsx

```diff
@@ -76,91 +78,140 @@ describe("MultiActionButton", () => {
       container = null;
     });
 
-    describe.each([
-      ["Enter", "Enter"],
-      ["Space", " "],
-      ["ArrowDown", "ArrowDown"],
-    ])("the %s key is pressed", (name, key) => {
-      it("then the first additional button should be focused", () => {
-        simulateBlur(mainButton);
-        wrapper.find(StyledMultiActionButton).simulate("mouseleave");
-        wrapper.update();
-        mainButton.simulate("keydown", { key });
-        jest.runAllTimers();
+    describe("using keyboard to open children container", () => {
+      it.each([
+        ["Enter", "Enter"],
+        ["Space", " "],
+        ["ArrowDown", "ArrowDown"],
```

Comment:
we should also test ArrowUp here, as per the requirements (see my comment on the whole PR)